### PR TITLE
data:Landouda

### PR DESCRIPTION
**Pull request by Landouda**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5643"></a>

### CODE REVIEW

File: data/Landouda.json
Comment (Panquesito7) [inline data/Landouda.json:13]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/Landouda.json
Comment (kumarsonsoff3) [inline data/Landouda.json:3]: Also, we no longer need the `displayStatsPublic` key. So please remove this from your `json`.

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by Landouda** [comment]
Thank you so much, i have pushed the fix and I'm waiting for your approval.

**Comment by Landouda** [comment]
I think there is an error while trying to merge my code by @SaraJaoude.
Error: Filename data/Landouda.json does not match user SaraJaoude.
Is it required for me to be the one doing the merge for the Data Filename Check hook to pass?

File: data/Landouda.json
Comment (eddiejaoude) [reply data/Landouda.json]: I think when merged, the GitHub Action will fix this

**Comment by SaraJaoude** [comment]
> I think there is an error while trying to merge my code by @SaraJaoude.
> Error: Filename data/Landouda.json does not match user SaraJaoude.
> Is it required for me to be the one doing the merge for the Data Filename Check hook to pass?

The GitHub action takes the latest commit and because I updated the branch it caused the action to fail, but this won't block your Pull Request.

**Pull request merged**
